### PR TITLE
[RHPAM-2813] Allow to use 'user' query parameter with /contents/output REST API.

### PR DESCRIPTION
**Pull request by elguardian**
kie api service in jbpm to store task event with another user.

### CODE REVIEW

**Comment by elguardian** [comment]
depends on https://github.com/kiegroup/droolsjbpm-knowledge/pull/435

File: jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/UserTaskService.java [hunk 1]
```diff
@@ -494,6 +494,15 @@
      */
     Long saveContent(Long taskId, Map<String, Object> values);
     
+    /**
+     * Saves gives values as content of a task, applies to task output as input cannot be altered
```
Comment (afalhambra) [inline jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/UserTaskService.java:498]: This method description might be reworded, as to differ saveContenFromUser from the saveContent method.

File: jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/UserTaskService.java [hunk 2]
```diff
@@ -494,6 +494,15 @@
...
+     * @param taskId
+     * @param values
```
Comment (afalhambra) [inline jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/UserTaskService.java:500]: Please, add the missing parameter "userId" in the doc.

```suggestion
     * @param taskId
     * @param userId
     * @param values
```

**Comment by elguardian** [comment]
@afalhambra can you review ?

**Pull request merged**